lib/app/features/chat/providers/conversation_message_actions_provider.c.dart

```diff
@@ -0,0 +1,238 @@
+// SPDX-License-Identifier: ice License 1.0
+
+import 'dart:convert';
+
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:ion/app/exceptions/exceptions.dart';
+import 'package:ion/app/features/auth/providers/auth_provider.c.dart';
+import 'package:ion/app/features/chat/model/entities/private_direct_message_data.c.dart';
+import 'package:ion/app/features/chat/model/entities/private_message_reaction_data.c.dart';
+import 'package:ion/app/features/chat/providers/conversation_message_management_provider.c.dart';
+import 'package:ion/app/features/core/providers/env_provider.c.dart';
+import 'package:ion/app/features/feed/data/models/bookmarks/bookmarks.c.dart';
+import 'package:ion/app/features/nostr/model/entity_expiration.c.dart';
+import 'package:ion/app/features/nostr/model/nostr_entity.dart';
+import 'package:ion/app/features/nostr/providers/nostr_event_signer_provider.c.dart';
+import 'package:ion/app/features/nostr/providers/nostr_notifier.c.dart';
+import 'package:ion/app/services/database/conversation_db_service.c.dart';
+import 'package:ion/app/services/logger/logger.dart';
+import 'package:ion/app/services/nostr/ion_connect_gift_wrap_service.c.dart';
+import 'package:ion/app/services/nostr/ion_connect_seal_service.c.dart';
+import 'package:nip44/nip44.dart';
+import 'package:nostr_dart/nostr_dart.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'conversation_message_actions_provider.c.g.dart';
+
+@Riverpod(keepAlive: true)
+Future<ConversationMessageActionsService> conversationMessageActionsService(
```

Comment:
I guess we don't need AsyncValue. Use `Future<Raw<ConversationMessageActionsService>>` as a return type.

Reply:
Done for all